### PR TITLE
[ISSUE #2079] Add unit test for ShenyuPluginPathBuilder 

### PR DESCRIPTION
**Pull request by hezhaoming**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-web/src/main/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilder.java [hunk 1]
```diff
@@ -31,15 +31,15 @@
  * The type Shenyu plugin path builder.
  */
 public final class ShenyuPluginPathBuilder {
-
```
Comment (yu199195) [inline shenyu-web/src/main/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilder.java:34]: donot modify this line

File: shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        File pluginPath = ShenyuPluginPathBuilder.getPluginPath("D:\\testpath");
+    }
+
+
+    /**
+     * test  for plugin-ext
+     * -Dplugin-ext=D:\testUrl
+     */
+    @Test
+    public void getPluginPath2() {
```
Comment (yu199195) [inline shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java:44]: pls add assert

**Review by yu199195 — APPROVED** [review]

File: shenyu-web/src/main/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilder.java (hunk 1 above)
Comment (hezhaoming) [reply shenyu-web/src/main/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilder.java:34]: ok

File: shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java (hunk 2 above)
Comment (hezhaoming) [reply shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java:46]: ok

**Comment by hezhaoming** [comment]
Resume format

File: shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        File pluginPath = ShenyuPluginPathBuilder.getPluginPath("");
+        Assert.assertNotNull(pluginPath);
+    }
+
+    /**
+     * test for default path
+     */
+    @Test
+    public void getPluginPath3() {
```
Comment (tydhot) [inline shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java:55]: plz don't name a method like this `****3`

Comment (hezhaoming) [reply shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java]: OK

File: shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+ */
+public final class ShenyuPluginPathBuilderTest {
+
+
+    /**
+     * test for  Custom path.
+     */
+    @Test
+    public void testGetPluginPathByCustomPath() {
+        File pluginPath = ShenyuPluginPathBuilder.getPluginPath("D:\\testpath");
```
Comment (tydhot) [inline shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java]: I am not sure it would be ok in linux or mac

**Review by yu199195 — APPROVED** [review]

**Pull request merged**